__tests__/app.test.js

```diff
@@ -39,3 +41,39 @@ describe.only('GET/api/topics', () => {
         })
     })
 })
+describe('GET/api/articles/:article_id', () => {
+    test('200 status code - returns user requested article as an array with object articles that contain all its properties', () => {
+        const articleId = 2
+        return request(app)
+        .get(`/api/articles/${articleId}`)
+        .expect(200)
+        .then(({body}) => {
+            const {article} = body;
+            expect({article: article}).toEqual(({article: {
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: 2,
+                    body: expect.any(String),
+                    topic: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number)
+                }})) 
+            })
+            })
+    })
+    test('404 code and sends an appropriate error message when given a valid but non-existent id', () => {
+        return request(app)
+        .get('/api/articles/999')
+        .expect(404)
+        .then((response) => {
```

Comment:
decide whether you want to destructure body from the response or not, and be consistent with your decision throughout your test suite